composer.json

```diff
@@ -40,7 +38,9 @@
         "aws/aws-sdk-php": "~2.7",
         "doctrine/mongodb-odm": "~1.0",
         "jackalope/jackalope-doctrine-dbal": "~1.1",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "symfony/phpunit-bridge": "~2.7|~3.0",
+        "friendsofsymfony/rest-bundle": "~1.1",
```

Comment:
Have a look at the composer docs (https://getcomposer.org/doc/04-schema.md#require-dev)

The dev packages are only loaded if you have them in your root ``composer.json``. No package of a child ``composer.json`` is loaded.